stable/influxdb/templates/ingress.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.ingress.enabled -}}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ template "influxdb.fullname" . }}
+  labels:
+    app: {{ template "influxdb.fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+  annotations:
+{{ toYaml .Values.ingress.annotations | indent 4 }}
+spec:
+{{- if .Values.ingress.tls }}
+  tls:
+    - hosts:
+      - {{ .Values.ingress.hostname | quote }}
+      secretName: {{ .Values.ingress.secretName | default (printf "%s-tls" (include "influxdb.fullname" .)) }}
```

Comment:
- This doesn't seem to make sense. Why would you want to use the secret for the DB credentials here?
- `secretName` should be `existingSecret`

Reply:
I was trying to reference some existing `ingress` setting like `chronograf` one 
https://github.com/kubernetes/charts/blob/fd93bff82fc18c33dc21e46848af9556c298af1d/stable/chronograf/templates/ingress.yaml#L18

This `secret` is for `ingress` TLS that shows in values.yaml below

Reply:
Forget about my comment regarding `secretName`. That's fine. But just blindly copying stuff doesn't make it right. It seems Chronograph is broken in that regard. It's fine if you just do `secretName: {{ .Values.ingress.secretName }}`.

Reply:
thanks for the feedback, removed the default value